wcivf/apps/results/views.py

```diff
@@ -0,0 +1,21 @@
+from django.views.generic import TemplateView
+
+from elections.models import Election
+from people.models import PersonPost
+
+class ResultsListView(TemplateView):
+    template_name = "results/results.list.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['elections'] = []
+        election_qs = Election.objects.filter(
+            current=True).order_by('election_date', '-election_weight')
```

Comment:
Actually South Derbyshire will still be shown top because it's on 11 May. Maybe instead we should do `.order_by('-election_date', '-election_weight')` to show the soonest first.